editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -643,7 +640,7 @@ void VersionControlEditorPlugin::_display_diff_split_view(List<EditorVCSInterfac
 			diff_line.old_text = line;
 			parsed_diff.push_back(diff_line);
 		} else if (diff_line.old_line_no == -1) {
-			int j = parsed_diff.size() - 1;
+			uint32_t j = parsed_diff.size() - 1;
```

Comment:
Am guessing `parsed_diff.size()` can never be zero here?

Reply:
Good ploiwill change